include/pstl/internal/unseq_backend_simd.h

```diff
@@ -29,464 +29,470 @@
+        return __result;
 
-    DifferenceType cnt = 1;
-    result[0] = first[0];
+    _DifferenceType __cnt = 1;
+    __result[0] = __first[0];
 
 __PSTL_PRAGMA_SIMD
-    for (DifferenceType i = 1; i < n; ++i) {
-__PSTL_PRAGMA_SIMD_ORDERED_MONOTONIC(cnt:1)
-        if (!pred(first[i], first[i - 1])) {
-            result[cnt] = first[i];
-            ++cnt;
+    for (_DifferenceType __i = 1; __i < __n; ++__i) {
+__PSTL_PRAGMA_SIMD_ORDERED_MONOTONIC(__cnt:1)
+        if (!__pred(__first[__i], __first[__i - 1])) {
+            __result[__cnt] = __first[__i];
+            ++__cnt;
         }
     }
-    return result + cnt;
+    return __result + __cnt;
 }
 
-template<class InputIterator, class DifferenceType, class OutputIterator, class Assigner>
-OutputIterator simd_copy_move(InputIterator first, DifferenceType n, OutputIterator result, Assigner assigner) noexcept {
+template<class _InputIterator, class _DifferenceType, class _OutputIterator, class _Assigner>
+_OutputIterator simd_copy_move(_InputIterator __first, _DifferenceType __n, _OutputIterator __result, _Assigner __assigner) noexcept {
 __PSTL_USE_NONTEMPORAL_STORES_IF_ALLOWED
 __PSTL_PRAGMA_SIMD
-    for (DifferenceType i = 0; i < n; ++i)
-        assigner(first + i, result + i);
-    return result + n;
+    for (_DifferenceType __i = 0; __i < __n; ++__i)
+        __assigner(__first + __i, __result + __i);
+    return __result + __n;
 }
 
-template<class InputIterator, class DifferenceType, class OutputIterator, class UnaryPredicate>
-OutputIterator simd_copy_if(InputIterator first, DifferenceType n, OutputIterator result, UnaryPredicate pred) noexcept {
-    DifferenceType cnt = 0;
+template<class _InputIterator, class _DifferenceType, class _OutputIterator, class UnaryPredicate>
+_OutputIterator simd_copy_if(_InputIterator __first, _DifferenceType __n, _OutputIterator __result, UnaryPredicate __pred) noexcept {
+    _DifferenceType __cnt = 0;
 
 __PSTL_PRAGMA_SIMD
-    for(DifferenceType i = 0; i < n; ++i) {
-        __PSTL_PRAGMA_SIMD_ORDERED_MONOTONIC(cnt:1)
-            if(pred(first[i])) {
-                result[cnt] = first[i];
-                ++cnt;
+    for(_DifferenceType __i = 0; __i < __n; ++__i) {
+        __PSTL_PRAGMA_SIMD_ORDERED_MONOTONIC(__cnt:1)
+            if(__pred(__first[__i])) {
+                __result[__cnt] = __first[__i];
+                ++__cnt;
             }
     }
-    return result + cnt;
+    return __result + __cnt;
 }
 
-template<class InputIterator, class DifferenceType, class BinaryPredicate>
-DifferenceType simd_calc_mask_2(InputIterator first, DifferenceType n, bool* __restrict mask, BinaryPredicate pred) noexcept {
-    DifferenceType count = 0;
+template<class _InputIterator, class _DifferenceType, class BinaryPredicate>
+_DifferenceType simd_calc_mask_2(_InputIterator __first, _DifferenceType __n, bool* __restrict __mask, BinaryPredicate __pred) noexcept {
+    _DifferenceType __count = 0;
 
 __PSTL_PRAGMA_SIMD_REDUCTION(+:count)
```

Comment:
There is an error "‘count’ has not been declared".
I suppose "__count" should be here.